BrewMesModulesParent/common/src/main/java/com/brewmes/common/services/ISubscribeService.java

```diff
@@ -0,0 +1,25 @@
+package com.brewmes.common.services;
+
+
+import com.brewmes.common.entities.MachineData;
+
+public interface ISubscribeService {
+
+    /**
+     * Sets up a subscribtion to the machine values based on a {@code connectionID} representing the machine.
+     * @param connectionID The id of the connection to subscribe to.
+     * @return {@code true} if the subscribtion went successfully, and {@code false} if something went wrong.
+     */
+ boolean subscribeToMachineValues(String connectionID);
+
+    /**
+     * Should return the latest {@code MachineData} measured.
```

Comment:
JavaDocs should be a description of functionality, not a proposal. 
Describe what the method is going to do in general terms.

Reply:
> Should return

It would be "returns"
There should be a general description. I propose something like.
```suggestion
     * Returns the latest reading from the specified machine via the {@code connectionID} argument.
```